EX04/index.js

```diff
@@ -0,0 +1,84 @@
+const express = require('express');
+const fs = require('fs');
+let app = express();
+
+
+app.get('/tasks', (req, res) => {
+    let printedTasks;
+    fs.readFile('tasks.json', (err,content) =>{
+        if(err) {
+            console.error(err);
+            return;
+        }
+        printedTasks = JSON.parse(content);
+        if(Object.keys(printedTasks).length === 0){
+            res.send("No tasks for you. Go to the beach");
```

Comment:
It is a good idea to return a clear error to the user (UX wise) but it should come with a valid HTTP status code (in this case 404 NOT FOUND)